src/api/index.ts

```diff
@@ -124,7 +124,7 @@ export function useApi(isOnline: boolean = true): BackendApi {
 			}
 
 			async function get(path: string, sessionAppToken?: string, sessionId?: number): Promise<AxiosResponse> {
-				return getWithLocalDbKey(path, sessionState?.id || sessionId.toString(), sessionAppToken);
+				return getWithLocalDbKey(path, sessionState?.id.toString() || sessionId.toString(), sessionAppToken);
```

Comment:
This [has `string` type already](https://github.com/wwWallet/wallet-frontend/blob/d80c9fc2045e2794760edec53b3a91a7fdca8178/src/api/index.ts#L19), is that insufficient somehow?

Reply:
Thant's the interesting.
In the case of `sessionState?.id`, if I am not mistaken, for someone reason we get type of number (i check it inside getWithLocalDbKey).

Reply:
Ok, then it probably happens [here](https://github.com/wwWallet/wallet-frontend/blob/d80c9fc2045e2794760edec53b3a91a7fdca8178/src/api/index.ts#L198) because `response.data` has `any` type, and therefore TypeScript just accepts any value at compilation time without adding any runtime type checks.

So I'd say we should rather put the `.toString()` there (and if we want to perform some actual type verification of the HTTP response data, that should happen there too).

Reply:
Hah, yep, we said the same thing at the same time :smile:

Reply:
Do you think there's anything else we need to do for now?

Reply:
Not that I know of.